mods/easy-emoticon.user.js

```diff
@@ -0,0 +1,146 @@
+        .replace(/\/star/, `${emoticonsCute[25]} `)
+        .replace(/\/flower/, `${emoticonsCute[26]} `)
+        .replace(/\/concern/, `${emoticonsCute[27]} `)
+        .replace(/\/4blush/, `${emoticonsCute[28]} `)
+        .replace(/\/3happy/, `${emoticonsCute[29]} `)
+        .replace(/\/4happy/, `${emoticonsCute[30]} `)
+        .replace(/\/unhappy/, `${emoticonsCute[31]} `)
+        .replace(/\/comfort/, `${emoticonsCute[32]} `)
+        .replace(/\/look/, `${emoticonsCute[33]} `)
+        .replace(/\/5happy/, `${emoticonsCute[34]} `)
+        .replace(/\/depress/, `${emoticonsCute[35]} `)
+        .replace(/\/shy/, `${emoticonsCute[36]} `)
+        .replace(/\/lenblush/, `${emoticonsCute[37]} `)
+        .replace(/\/heh/, `${emoticonsCute[38]} `)
+        .replace(/\/energy/, `${emoticonsCute[39]} `)
+        .replace(/\/1welp/, `${emoticonsCute[40]} `)
+        .replace(/\/plead/, `${emoticonsCute[41]} `)
+        .replace(/\/glad/, `${emoticonsCute[42]} `)
+        .replace(/\/stoic/, `${emoticonsCute[43]} `)
+        .replace(/\/cheer/, `${emoticonsCute[44]} `)
+        .replace(/\/rush/, `${emoticonsCute[45]} `)
+        .replace(/\/um/, `${emoticonsCute[46]} `)
+        .replace(/\/scared/, `${emoticonsCute[47]} `)
+        .replace(/\/evil/, `${emoticonsCute[48]} `)
+        .replace(/\/sparkles/, `${emoticonsCute[49]} `)
+        .replace(/\/bash/, `${emoticonsCute[50]} `)
+        .replace(/\/blank/, `${emoticonsCute[51]} `)
+        .replace(/\/sus/, `${emoticonsCute[52]} `)
+        .replace(/\/2heh/, `${emoticonsCute[53]} `)
+        .replace(/\/smug/, `${emoticonsCute[54]} `)
+        .replace(/\/3welp/, `${emoticonsCute[55]} `)
+        .replace(/\/nunnun/, `${emoticonsCute[56]} `)
+        .replace(/\/squint/, `${emoticonsCute[57]} `)
+        .replace(/\/donger/, `${emoticonsCute[58]} `)
+        .replace(/\/2donger/, `${emoticonsCute[59]} `)
+        .replace(/\/3donger/, `${emoticonsCute[60]} `)
+        .replace(/\/4donger/, `${emoticonsCute[61]} `)
+        .replace(/\/5donger/, `${emoticonsCute[62]} `)
+        .replace(/\/6donger/, `${emoticonsCute[63]} `)
+        .replace(/\/7donger/, `${emoticonsCute[64]} `)
+        .replace(/\/hmpf/, `${emoticonsCute[65]} `)
+        .replace(/\/2awesome/, `${emoticonsCute[66]} `)
+        .replace(/\/2blank/, `${emoticonsCute[67]} `)
+        .replace(/\/3blank/, `${emoticonsCute[68]} `)
+        .replace(/\/4blank/, `${emoticonsCute[69]} `)
+        .replace(/\/5blank/, `${emoticonsCute[70]} `)
+        .replace(/\/2um/, `${emoticonsCute[71]} `)
+        .replace(/\/2cheer/, `${emoticonsCute[72]} `)
+        .replace(/\/patpat/, `${emoticonsCute[73]} `)
+        .replace(/\/dance/, `${emoticonsCute[74]} `)
+        .replace(/\/why/, `${emoticonsCute[75]} `)
+        .replace(/\/6blank/, `${emoticonsCute[76]} `)
+
+        ;
+
+    param.value = text;
+
+}
+
+function easyEmoticon(toggle) {
+    if (toggle) {
+        emoticonGen();
+    }
```

Comment:
I think you should look at setting a global variable and setting it to your event listener inside emoticonGen(). This would allow you to remove the listener when toggled off.

Reply:
done, thanks for the advice!